doc/enterprise/deployment.md

```diff
@@ -0,0 +1,117 @@
+
+NAME             CLUSTER-IP   EXTERNAL-IP   PORT(S)                         AGE
+svc/dash         10.0.0.4     <nodes>       8080:30080/TCP,8081:30081/TCP   8m
+svc/etcd         10.0.0.152   <nodes>       2379:32379/TCP                  13m
+svc/kubernetes   10.0.0.1     <none>        443/TCP                         17m
+svc/pachd        10.0.0.193   <nodes>       650:30650/TCP,651:30651/TCP     13m
+
+NAME           DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deploy/dash    1         1         1            1           8m
+deploy/etcd    1         1         1            1           13m
+deploy/pachd   1         1         1            1           13m
+
+NAME                  DESIRED   CURRENT   READY     AGE
+rs/dash-3809689541    1         1         1         8m
+rs/etcd-4197107720    1         1         1         13m
+rs/pachd-4280389576   1         1         1         13m
+```
+
+If you previously had port forwarding enabled for your Pachyderm cluster, you will also have to restart this forwarding:
+
+```
+$ ps aux | grep "forward"
+dwhitena 12625  0.0  0.3 475896 26204 pts/13   SNl  14:51   0:00 pachctl port-forward
+dwhitena 12674  0.0  0.4  58232 35208 pts/13   SNl  14:51   0:00 kubectl port-forward pachd-4280389576-qfcg8 30650:650
+dwhitena 15467  0.0  0.0  14224  1040 pts/13   S+   15:06   0:00 grep --color=auto --exclude-dir=.bzr --exclude-dir=CVS --exclude-dir=.git --exclude-dir=.hg --exclude-dir=.svn forward
+$ kill 12674
+Terminated                                                                                                                                                                                                              
+UI not enabled, deploy with --dashboard
+[1]  + 12625 exit 1     pachctl port-forward
+$ pachctl port-forward &
+[1] 15481
+Pachd port forwarded
+Dash websocket port forwarded
+Dash UI port forwarded, navigate to localhost:38080
+CTRL-C to exit
+```
+
+Now you can visit the Pachyderm dashboard at `localhost:38080`!  The dashboard will prompt you for your enterprise token that you received when registering for Pachyderm Enterprise Edition:
+
+![alt tag](token.png)
+
+## Deploying with a new Pachyderm deployment
+
+You can deploy Pachyderm Enterprise Edition with any new Pachyderm deployment by adding the `--dashboard` flag to the respective deploy command:
+
+```
+# AWS
+pachctl deploy amazon ... --dashboard
+
+# Google
+pachctl deploy google ... --dashboard
+
+# Azure
+pachctl deploy azure ... --dashboard
+
+# Local
+pachctl deploy local --dashboard
+
+# Custom
+pachctl dpeloy custom ... --dashboard
+```
+
+Each of these deploys are further details [here](http://pachyderm.readthedocs.io/en/latest/deployment/deploy_intro.html).  Once deploying with those commands, you should see the "dash" image running in Kubernetes, and you should be able to access the dashboard at `localhost:38080`, as discussed further in [above](#deploying-on-top-of-an-existing-pachyderm-deployment).
```

Comment:
the dash address changes to localhost:30080 in the next release (1.5.1; change is already merged on master)